automation-swiftui/AutomationAssessment/ContentView.swift

```diff
@@ -55,39 +49,38 @@ struct ContentView: View {
                 .font(.subheadline)
         }
     }
-
+    
     var headerView: some View {
-        VStack {
+        VStack(alignment: .leading) {
```

Comment:
This fixes the alignment issue with the text- it was being aligned centrally here, but leading in 'subtitleView' so it looked strange with a gap at the beginning